### PR TITLE
Release msal-node-extensions 1.0.0-alpha.0

### PR DESCRIPTION
**Pull request by sangonzal**

### CODE REVIEW

File: samples/msal-node-extensions/index.js [hunk 1]
```diff
@@ -65,3 +65,26 @@ extensions.FilePersistenceWithDataProtection.create(cachePath, extensions.DataPr
...
         }
     );
+
+/**
+ * Builds persistence based on operating system. Falls back to storing in plain text.
+ */
+async function createPersistence(){
+    // On Windows, uses a DPAPI encrypted file
+    if(process.platform === "win32"){
+        return extensions.FilePersistenceWithDataProtection.create(cachePath, extensions.DataProtectionScope.LocalMachine)
```
Comment (jo-arroyo) [inline samples/msal-node-extensions/index.js:75]: Nit:

```suggestion
        return extensions.FilePersistenceWithDataProtection.create(cachePath, extensions.DataProtectionScope.LocalMachine);
```

File: samples/msal-node-extensions/package.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
   "author": "Microsoft",
   "license": "MIT",
   "dependencies": {
-    "@azure/msal-node": "^0.1.0",
-    "express": "^4.17.1",
-    "@azure/msal-node-extensions": "0.1.0"
+    "@azure/msal-node": "^1.0.0-alpha.0",
+    "@azure/msal-node-extensions": "1.0.0-alpha.0",
+    "express": "^4.17.1"
```
Comment (jasonnutter) [inline samples/msal-node-extensions/package.json:16]: Be sure to commit the updated `package-lock.json` from this sample as well.

Comment (sangonzal) [reply samples/msal-node-extensions/package.json:16]: Yep, will do.

Comment (sangonzal) [reply samples/msal-node-extensions/package.json]: Looks like the package.json and package-lock.json were both updated in your CodeQL PR, so there were was no need to update here.

**Pull request merged**